Engines/Wine/Plugins/virtual desktop/script.js

```diff
@@ -7,13 +7,20 @@ include("engines.wine.engine.object");
  * @returns {Wine} Wine object
  */
 Wine.prototype.setVirtualDesktop = function (width, height) {
-    var regeditFileContent =
+    if (!width)
+        const width = getScreenWidth();
```

Comment:
Remove the `const` otherwise you will get an error because you're trying to redefine an existing variable